03-task-manager/starter/controllers/tasks.js

```diff
@@ -0,0 +1,51 @@
+const Task = require('../models/Task')
+
+const getAllTasks = async (req, res) => {
+    const tasks = await Task.find({})
+    res.status(200).json({tasks})
+  }
+
+const createTask = async (req, res) => {
+    const task = await Task.create(req.body)
+    //res.send('Create task')
+    res.status(201).json({task})
+}
+
+const getTask = async (req, res, next) => {
+    const { id: taskID } = req.params
+    const task = await Task.findOne({ _id: taskID })
+    if (!task) {
+      return next(createCustomError(`The task with id : ${taskID}, does not exist`, 404))
```

Comment:
The createCustomerError is not defined, and the app crashes when the task is not found. 
You can return a 404 response here. Like this:
`return res.status(404).json({success: false, message:`The task with id : ${taskID}, does not exist`})`